actors/builtin/miner/miner_actor.go

```diff
@@ -526,9 +482,10 @@ func (a Actor) DeclareTemporaryFaults(rt Runtime, params DeclareTemporaryFaultsP
 	builtin.RequireSuccess(rt, code, "failed to burn fee")
 
 	// Request deferred Cron invocation to update temporary fault state.
+	// TODO: cant we just lazily clean this up?
```

Comment:
When we change to the list of faultdecls stored in state, I think we'll be in much better shape to iterate that lazily.